src/core/cgroup.c

```diff
@@ -3999,6 +3999,16 @@ static uint64_t unit_get_effective_limit_one(Unit *u, CGroupLimitType type) {
                 [CGROUP_LIMIT_TASKS_MAX]   = offsetof(CGroupContext, tasks_max),
         };
 
+        /* These callbacks return system global or container's root cgroup attributes */
+        static uint64_t (*callbacks[_CGROUP_LIMIT_MAX])(void) = {
+                [CGROUP_LIMIT_MEMORY_MAX]  = &physical_memory,
+                [CGROUP_LIMIT_MEMORY_HIGH] = &physical_memory,
+                [CGROUP_LIMIT_TASKS_MAX]   = &system_tasks_max,
```

Comment:
the `&` before function names is redundant, and we never use it otherwise, hence please don#t use it here.